src/coreclr/tools/Common/TypeSystem/Interop/UnmanagedCallingConventions.cs

```diff
@@ -135,6 +135,25 @@ public static UnmanagedCallingConventions GetPInvokeMethodCallingConventions(thi
             return result;
         }
 
+        public static UnmanagedCallingConventions GetDelegateCallingConventions(this TypeDesc delegateType)
```

Comment:
This method doesn't appear used. Is that intentional?

Reply:
I removed the usage of it during a refactor. I think it's still useful to have, but we can remove it and re-add later when we have a use case.